src/lib/input/input.spec.ts

```diff
@@ -515,36 +516,36 @@ describe('MdInput without forms', function () {
     fixture.detectChanges();
 
     let inputEl = fixture.debugElement.query(By.css('input')).nativeElement;
-    let labelEl = fixture.debugElement.query(By.css('label')).nativeElement;
+    let formFieldEl = fixture.debugElement.query(By.css('.mat-form-field')).nativeElement;
```

Comment:
This sounds like something we should do as part o a separate cleanup pass of the whole project, I'm sure there are tools that can analyze and fix this for us